js/src/record/helpers/getArweaveRecord.ts

```diff
@@ -0,0 +1,13 @@
+import { Connection } from "@solana/web3.js";
+import { Record } from "../../types/record";
+import { getRecord } from "../getRecord";
+
+/**
+ * This function can be used to retrieve the Arweave record of a domain name
+ * @param connection The Solana RPC connection object
+ * @param domain The .sol domain name
+ * @returns
+ */
+export const getArweaveRecord = (connection: Connection, domain: string) => {
+  return getRecord(connection, domain, Record.ARWV, true);
+};
```

Comment:
@dr497 previously each `getRecord` function was wrapped in `async` and returned a value like `return await getRecord`, for example like [here](https://github.com/Bonfida/sns-sdk/blob/feat/js-sdk-improvement/js/src/record.ts#L301-L303). I think it was an overhead so I removed it.